listenbrainz/webserver/static/js/src/APIService.ts

```diff
@@ -119,6 +119,30 @@ export default class APIService {
     return { status: response.status };
   };
 
+  getFollowers = async (username: string) => {
+    if (!username) {
+      throw new SyntaxError("Username missing");
```

Comment:
this should be something equivalent to valueerror, i guess. SyntaxError seems incorrect.